ktorm-support-mysql/src/main/kotlin/me/liuwj/ktorm/support/mysql/MySqlDialect.kt

```diff
@@ -191,9 +211,8 @@ open class MySqlExpressionVisitor : SqlExpressionVisitor() {
         }
     }
 
-    protected open fun visitBulkInsertAssignments(
-        assignments: List<List<ColumnAssignmentExpression<*>>>
-    ): List<List<ColumnAssignmentExpression<*>>> {
+    protected open fun visitBulkInsertAssignments(assignments: List<List<ColumnAssignmentExpression<*>>>):
```

Comment:
Please use the existing code style, do not change it.

Reply:
OK